SwiftWisdom/Core/UIKit/UIView+Extensions.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIView+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Alexander Persian on 12/29/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+
+    /// Zooms a view by scaling using the provided factor.
+    func ip_zoom(by factor: CGFloat) {
+        transform = CGAffineTransform(scaleX: factor, y: factor)
```

Comment:
I believe this does use the original transform. It could actually be replaced by `transform.scaledBy(x: factor, y: factor)` to be more clear?